index.html

```diff
@@ -1,9 +1,20 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hangman</title>
-  </head>
-  <body>
-    <script src="hangman.js" type="text/javascript"></script>
-  </body>
+<head>
+  <title>Hangman</title>
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js" type="text/javascript"></script>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <div id="out" class="output">
+  </div>
+  <div class="input">
+    <p>&lt;</p>
+    <textarea id="in" onkeypress="if(event.keyCode == 13) ProcIn();" autofocus></textarea>
```

Comment:
recommend moving logic out of your html if possible. In general we prefer the vanilla `addEventListener`, and in this case listen for a `keydown` event, and if that keycode is 13, then call the `ProcIn` method